cypress/component/Navbar.cy.tsx

```diff
@@ -0,0 +1,11 @@
+import Navbar from "../../src/components/Navbar"
+import withProviders from "../../src/hoc/withProviders"
+
+const NavbarWithProviders = withProviders(Navbar)
+
+
+describe('Cypress', () => {   
```

Comment:
takie rzeczy właśnie fajnie się załatwia zwykłym unit testem, ale oczywiście nie jest błędem tworzenie automatu na to :D
Jeśli chodzi o e2e w takim przypadku to można sprawdzić zmienianie flag - wejście na stronę, znalezienie i sprawdzenie domyślnego języka - zmiana na drugi język - upewnienie się, że wszystko gra - zmiana na trzeci - znowu sprawdzenie poprawności itd

Reply:
Ten test powstał tuż po zainstalowaniu cypressa jak sobie sprawdzałem co i jak. Potem skupiłem się już na testach w folderze e2e, a tego zapomniałem usunąć :D